packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/summary/bank-account/bank-account.html

```diff
@@ -0,0 +1,98 @@
+<h3
+    data-translate="telephony_billingAccount_svaWallet_kyc_summary_bankaccount_title"
+></h3>
+
+<div class="oui-field oui-field_auto">
+    <div class="oui-field__header">
+        <label
+            class="oui-field__label"
+            data-translate="telephony_billingAccount_svaWallet_kyc_identity_bankAccount_holder"
+        ></label>
+    </div>
+    <div
+        class="oui-field__control"
+        data-ng-bind="$ctrl.bankAccount.holder"
+    ></div>
```

Comment:
The `div` is a structural element, to bind text i suggest to you to use one of those: `<p>`, `<dl><dt>...</dl>/<dt>` or a `<span>`.
```suggestion
    <div
        class="oui-field__control"
    >
        <p data-ng-bind=":: $ctrl.bankAccount.holder"></p>
    </div>
```